pxtrunner/tsconfig.json

```diff
@@ -3,10 +3,19 @@
         "target": "es2017",
         "noImplicitAny": true,
         "noImplicitReturns": true,
-        "declaration": true,
-        "outFile": "../built/pxtrunner.js",
+        "noImplicitThis": true,
+        "module": "commonjs",
+        "outDir": "../built/pxtrunner",
+        "rootDir": "..",
         "newLine": "LF",
         "sourceMap": false,
+        "moduleResolution": "node",
+        "isolatedModules": false,
+        "jsx": "react-jsx",
```

Comment:
nope, i just copied the tsconfig for the webapp

Reply:
This might be a bit nitpicky, but is this jsx part needed? Jsx is only used for components, right? Is it needed in a module that doesn't have components associated with it?

Reply:
removed!